centreon/src/App/MonitoringConfiguration/Infrastructure/Security/InstallationCommandVoter.php

```
@@ -0,0 +1,74 @@
+
+/**
+ * @extends Voter<value-of<AgentConfigurationPermissionEnum>, mixed>
+ */
+final class InstallationCommandVoter extends Voter
+{
+    public function __construct(
+        private readonly ResourceAccessRepository $resourceAccessRepository,
+    ) {
+    }
+
+    protected function supports(string $attribute, mixed $subject): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$subject'. (undefined)

(UnusedFormalParameter)